redbaron/base_nodes.py

```diff
@@ -207,10 +207,27 @@ def absolute_bounding_box(self):
 
     def find_by_position(self, position):
         path = Path.from_baron_path(self, baron.path.position_to_path(self.fst(), position))
-        if path:
-            return path.node
-        else:
-            return None
+        return path.node if path else None
+
+    def at(self, line_no):
+        if not 0 <= line_no <= self.absolute_bounding_box.bottom_right.line:
+            raise IndexError("Line number {0} is outside of the file".format(line_no))
+        node = self.find_by_position((line_no, 1))
+        if node.absolute_bounding_box.top_left.line == line_no:
+            if hasattr(node.parent, 'absolute_bounding_box') and \
+                            node.parent.absolute_bounding_box.top_left.line == line_no and \
+                    node.parent.parent is not None:
```

Comment:
@ibizaman, thank you for response!
I think no. Because this if clause is for ```EndlNode``` only. Since using anything from ```nodes``` module is not appropriate due to cross importing, I have used this checking. 
Please fix me if I am wrong. It will be great to see test case from you.